pytom/gpu/gpuStructures.py

```diff
@@ -121,6 +121,8 @@ def template_matching_gpu(self):
         CX, CY, CZ = SX // 2, SY // 2, SZ // 2
         cx, cy, cz = sx // 2, sy // 2, sz // 2
         mx, my, mz = sx % 2, sy % 2, sz % 2
+        # rotation center needs to be set, voltools uses (s - 1) / 2 by default, but pytom_volume s // 2 - s % 2
+        rotation_center = (cx + mx, cy + my, cz + mz)
```

Comment:
either the comment is wrong or the sign on these:
```suggestion
        # rotation center needs to be set, voltools uses (s - 1) / 2 by default, but pytom_volume s // 2 + s % 2
        rotation_center = (cx + mx, cy + my, cz + mz)
```
or
```suggestion
        # rotation center needs to be set, voltools uses (s - 1) / 2 by default, but pytom_volume s // 2 - s % 2
        rotation_center = (cx - mx, cy - my, cz - mz)
```